planet/src/main/webapp/js/placesItinerary.js

```diff
@@ -97,18 +181,14 @@ async function submitPlace(ref) {
     const eventListSnapshot = await eventListRef.once('value');
     const order = eventListSnapshot.numChildren() + 1;
 
-    // Create a new event based on the place
-    const placeRef = database.ref('places/' + ref); 
-    const placeSnapshot = await placeRef.once('value');
-    const placeDetails = placeSnapshot.val();
-
-    const newEventRef = eventListRef.child(ref);
+    const place = savedPlaces[placeId];
```

Comment:
We should probably check for null in this case. I don't think it's possible for it to be null, but let's at least handle the error gracefully if it is (maybe a console log and skipping the rest of the steps?

Reply:
Okay!